backend/census_historical_migration/workbooklib/notes_to_sefa.py

```diff
@@ -189,6 +195,30 @@ def xform_missing_notes_records(audit_header, policies_content, rate_content):
     return policies_content, rate_content
 
 
+def transform_missing_note_title_and_content(notes):
+    """Transforms missing note title and note content."""
```

Comment:
This is adding both title and content to any note record without title and content. What we want to do instead is to:
-  Add title only if content exists and title is missing.
- Add content only if title exists and content is missing.

It is valid to have a note record without both title and content but it is not valid to have a note record with only one (title without content or content without title)